src/komponenter/header/header-regular/common/varsler/varsel-boks/VarselBoks.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import { Next } from '@navikt/ds-icons';
+import Tekst from 'tekster/finn-tekst';
+import { fjernLestVarsel } from 'store/reducers/varselinnboks-duck';
+import { useDispatch } from 'react-redux';
+import { postDone } from 'api/api';
+import { logAmplitudeEvent } from 'utils/analytics/amplitude';
+import ArkiverKnapp from './arkiver-knapp/ArkiverKnapp';
+import classNames from 'classnames';
+import style from './VarselBoks.module.scss';
+
+type Props = {
+    eventId: string;
+    apiVarselinnboksUrl: string;
+    tekst: string;
+    dato: string;
+    href: string;
+    isMasked: boolean;
+    id?: string;
+    type: string;
+    setActivateScreenReaderText: (setActivateScreenReaderText: boolean) => void;
+};
+
+const Beskjed = ({
+    eventId,
+    apiVarselinnboksUrl,
+    tekst,
+    dato,
+    href,
+    isMasked,
+    id,
+    type,
+    setActivateScreenReaderText,
+}: Props) => {
+    //TODO: Legge inn stepup-tekst i alle språk.
+    const [isHover, setIsHover] = useState(false);
+
+    const dispatch = useDispatch();
+
+    const hasNoHref = (href: string) => href === undefined || href === null || href === '';
+    const isOppgave = type === 'OPPGAVE';
+    const isArkiverbar = (href: string) => hasNoHref(href) && type !== 'OPPGAVE';
```

Comment:
Kan kanskje forenkles litt:
```suggestion
    const isArkiverbar = !href && !isOppgave;
```